android-app/app/src/main/java/de/dhbw/ka/se/fibo/ui/dashboard/DashboardFragment.java

```diff
@@ -176,7 +277,7 @@ private void setDateCardTime() {
 
             DateTimeFormatter formatter = builder.toFormatter(Locale.getDefault());
 
-            binding.dateStartEndText.setText(requireContext().getString(R.string.timespanStartToEnd, formatter.format(startDate), formatter.format(endDate)));
+            binding.dateStartEndText.setText(context.getString(R.string.timespanStartToEnd, formatter.format(startDate), formatter.format(endDate)));
         } else {
             Log.w("FiBo", "setDateCardTime(): startDate != endDate");
```

Comment:
I meant this one
```suggestion
            binding.dateStartEndText.setText(context.getString(R.string.timespanStartToEnd, formatter.format(startDate), formatter.format(endDate)));
        } else {```

Reply:
this code wasn't changed and this is something that shouldn't happen and hence it is appropiate to do a log in the warning-level.